fragments/master.sh

```diff
@@ -252,8 +248,24 @@ wait_on_api_server(){
         echo "Error: API service cannot be reched ..."
   fi
 }
+
+set_docker_registry(){
+  DOCKER_CONF="/usr/lib/systemd/system/docker.service"
```

Comment:
Yes it works only for centos. I will add if condition to execute it only for centos. It will be more safe.